app.py

```diff
@@ -49,141 +71,35 @@ def webhook():
-            names.sort()
-            wins = [int(i) for i in standings.groupby('Name').sum().reset_index()['Wins'].tolist()]
-            losses = [int(i) for i in standings.groupby('Name').sum().reset_index()['Losses'].tolist()]
-            message = str()
-            for i in range(0, len(names)):
-                message += names[i] + ': ' + str(wins[i]) + '-' + str(losses[i]) + '\n'
-            print(message)
-
+            message = get_standings_message(standings)
             return send_message(message)
 
         elif currentmessage == 'mother fucking standings':
-            names = standings['Name'].unique().tolist()
-            names.sort()
-            wins = [int(i) for i in standings.groupby('Name').sum().reset_index()['Wins'].tolist()]
-            losses = [int(i) for i in standings.groupby('Name').sum().reset_index()['Losses'].tolist()]
-            message = str()
-            for i in range(0, len(names)):
-                message += names[i] + ': ' + str(wins[i]) + '-' + str(losses[i]) + '\n'
-            print(message)
-
+            message = get_standings_message(standings)
             return send_message(message.upper())
 
         # Message options - either all teams, a player's teams, or print help
-        elif currentmessage == "patrick teams":
-            return_contestant('Patrick')
-        elif currentmessage == "jordan teams":
-            return_contestant('Jordan')
-        elif currentmessage == "nathan teams":
-            return_contestant('Nathan')
-        elif currentmessage == "jack teams":
-            return_contestant('Jack')
-        elif currentmessage == "all teams":
-            return_contestant('All')
+
+        elif len(currentmessage.split()) == 2 and currentmessage.split()[-1] == 'teams':
+            name = currentmessage.split()[0].capitalize()
+            if name in ['Jack', 'Jordan', 'Patrick', 'Nathan', 'All']:
+                return_contestant(name, standings)
+            elif name == 'My':
+                return_contestant(groupme_users[currentuser].split()[0], standings)
         elif currentmessage == 'nfl bot help':
             options = configs.base_configs['Responses']
             header = "Input options for the NFL Wins Tracker bot:\n"
             message = header + "\n".join(options)
             return send_message(message)
-        elif currentmessage == 'my teams':
-            return_contestant(groupme_users[currentuser].split()[0])
     elif currentmessage[:4].lower() == '!who':
-        jack_teams = configs.base_configs['Jack']
-        jordan_teams = configs.base_configs['Jordan']
-        nathan_teams = configs.base_configs['Nathan']
-        patrick_teams = configs.base_configs['Patrick']
+        # I think this can be teams_list = [get_teams()] but will test later
+        jack_teams, jordan_teams, nathan_teams, patrick_teams = get_teams()
         teams_list = [jack_teams, jordan_teams, nathan_teams, patrick_teams]
```

Comment:
We might want to consider making this a dictionary at some point rather than creating a bunch of lists.
So then it would just be like `{"Jack": ["Cardinals", "Bears", ...], "Jordan": ...}`, and then we can reference them like so: `team_dict["Jack"]`

Reply:
Yep, definitely a good idea. I'll leave this for now, but I think that's definitely the direction we should go